docs/api.md

```diff
@@ -57,6 +57,27 @@ number.tryParse("404");
 // => 404
 ```
 
+### bnb.all(parsers)
+
+Parses all parsers in order, returning the values in the same order.
+
+**Note:** The parsers do not all have to return the same type.
+
+```ts
+const abc = bnb
+  .all([bnb.string("a"), bnb.string("b"), bnb.string("c")])
```

Comment:
Shouldn't it be `bnb.text` instead of `bnb.string`

Reply:
Old Parsimmon habit 😆  Thank you!